packages/fe-container/src/components/top-bar/TopBar.tsx

```diff
@@ -16,6 +16,8 @@
 import React, {useContext} from 'react'
 
 import {BurgerIcon} from '@components/burger-icon/BurgerIcon'
+import {Divider} from 'antd'
+import {HelpIcon} from '@components/help-icon/HelpIcon'
```

Comment:
antd is an external library, this is not the right place to import it :smile_cat: